.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: latest next podman-next ${{ github.sha }}
+  IMAGE_REGISTRY: quay.io/podman
+  COPR_OWNER: rhcontainerbot
+  COPR_PROJECT: podman-next
+
+jobs:
+  fcos-podman-next-image-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for successful podman-next build with the latest commit
+      run: |
+        # TODO: add this in the Containerfile itself or as a --build-arg
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $(git rev-parse --short ${{ github.sha }})
+        echo "podman-next build successful."
+
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Build FCOS Image
+      id: build_image
+      # Ref: https://github.com/redhat-actions/buildah-build
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
+        tags: ${{env.IMAGE_TAGS }}
+        containerfiles: |
+          ./contrib/podman-next/fcos-podmanimage/Containerfile .
```

Comment:
Minor-nit: There's no need to do the `|` line-wrapping thing for a single line.

Reply:
that's what the redhat-action doc used, so I used the same. I can remove it in followup